apps/gallery/js/thumbnails.js

```diff
@@ -1,12 +1,25 @@
 /* exported Thumbnails */
 /* global ThumbnailList, ThumbnailDateGroup, MediaDB, LazyLoader */
-/* global photodb, files, picking, metadataParser */
+/* global photodb, files, picking, metadataParser, IntlHelper */
 (function(exports) {
   'use strict';
 
   // This is the object we export
   var thumbnails = exports.Thumbnails = {};
 
+  IntlHelper.define('date-group', 'datetime', {
+    month: 'long',
+    year: 'numeric',
+  });
+
+  IntlHelper.define('time-stamp', 'datetime', {
+    hour: 'numeric',
+    minute: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric'
+  });
+
```

Comment:
Since date-group is used by thumbnail_group.js and time-stamp is used by thumbnail_item.js maybe it would be better to manage these in those files rather than here?